src/lC_common_h.jl

```diff
@@ -1,5 +1,5 @@
 macro c(ret_type, func, arg_types, lib)
-  local args_in = Any[ symbol(string('a',x)) for x in 1:length(arg_types.args) ]
+  local args_in = Any[Symbol(string('a',x)) for x in 1:length(arg_types.args)]
```

Comment:
Is this better?
`local args_in = [symbol('a',i) for i in eachindex(arg_types.args)]`